.buildkite/check-performance.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
Do you think we could do this using something like `pytest-benchmark` https://pypi.org/project/pytest-benchmark/ instead of a bash script?

Reply:
not familiar with it, but I'd say not here, I have a fair amount of git bookeeping to do here